docs/man/docker-ps.1.md

```diff
@@ -37,6 +37,8 @@ the running containers.
    Provide filter values. Valid filters:
                           exited=<int> - containers with exit code of <int>
                           status=(restarting|running|paused|exited)
+                          name=a_name_to_match
```

Comment:
I think it would be more consistent to use either `<string>` or `<name> - containers with a name matching <name>` here